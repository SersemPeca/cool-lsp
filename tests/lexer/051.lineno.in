(* comment (* "

" *) nested comment end
*)
!
bond
""
