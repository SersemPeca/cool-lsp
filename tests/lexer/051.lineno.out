#5 ERROR: Invalid symbol "!"
#6 OBJECTID bond
#7 STR_CONST ""
